docker/nginx/conf.d/client.conf

```diff
@@ -41,39 +41,102 @@ server {
 	client_body_buffer_size 128k;
 	client_max_body_size 128k;
 
-	rewrite "^(/([a-zA-Z0-9-_]{46})[^/]{0})$" $1/ permanent;
-	rewrite "^(/hns/[^/]+)[^/]{0}$" $1/ permanent;
+	location / {
+		include /etc/nginx/conf.d/include/cors;
+
+		root /var/www/webportal;
+	}
 
 	location /blacklist {
+		include /etc/nginx/conf.d/include/cors;
+
 		proxy_cache skynet;
 		proxy_cache_valid any 1m; # cache blacklist for 1 minute
+		proxy_set_header User-Agent: Sia-Agent;
+		proxy_pass http://siad-download/skynet/blacklist;
+	}
+
+	location /skynet/blacklist {
+		include /etc/nginx/conf.d/include/cors;
+
+		proxy_cache skynet;
+		proxy_cache_valid any 1m; # cache blacklist for 1 minute
+		proxy_set_header User-Agent: Sia-Agent;
 		proxy_pass http://siad-download/skynet/blacklist;
 	}
 
 	location /portals {
+		include /etc/nginx/conf.d/include/cors;
+
 		proxy_cache skynet;
 		proxy_cache_valid any 1m; # cache portals for 1 minute
+		proxy_set_header User-Agent: Sia-Agent;
+		proxy_pass http://siad-download/skynet/portals;
+	}
+
+	location /skynet/portals {
+		include /etc/nginx/conf.d/include/cors;
+
+		proxy_cache skynet;
+		proxy_cache_valid any 1m; # cache portals for 1 minute
+		proxy_set_header User-Agent: Sia-Agent;
 		proxy_pass http://siad-download/skynet/portals;
 	}
 
 	location /stats {
+		include /etc/nginx/conf.d/include/cors;
+
+		proxy_cache skynet;
+		proxy_cache_valid any 1m; # cache stats for 1 minute
+		proxy_set_header User-Agent: Sia-Agent;
+		proxy_pass http://siad-upload/skynet/stats;
+	}
+
+	location /skynet/stats {
+		include /etc/nginx/conf.d/include/cors;
+
 		proxy_cache skynet;
 		proxy_cache_valid any 1m; # cache stats for 1 minute
+		proxy_set_header User-Agent: Sia-Agent;
 		proxy_pass http://siad-upload/skynet/stats;
 	}
 
 	location /statsdown {
+		include /etc/nginx/conf.d/include/cors;
+
 		proxy_cache skynet;
 		proxy_cache_valid any 1m; # cache stats for 1 minute
+		proxy_set_header User-Agent: Sia-Agent;
 		proxy_pass http://siad-download/skynet/stats;
 	}
 
+	location /health-check {
+		include /etc/nginx/conf.d/include/cors;
+
+		proxy_pass http://health-check:3100;
```

Comment:
Heathcheck mapped on the same port as Handshake?

Reply:
yeah, it's different ip so it's ok